packages/ngx-web-component/package.json

```diff
@@ -16,12 +16,12 @@
     },
     "private": false,
     "nx": {
-        "implicitDependencies": ["@readalongs/web-component"]
+        "implicitDependencies": ["web-component"]
```

Comment:
ARGGGGG

Reply:
Loud enough for you to hear me all the way from Ottawa!